src/main/java/io/pivotal/cfapp/controller/LegacyWorkloadsController.java

```diff
@@ -28,16 +36,24 @@ public LegacyWorkloadsController(
         this.util = new TkServiceUtil(tkService);
     }
 
+
     @GetMapping(value = { "/snapshot/detail/legacy" } )
-	public Mono<ResponseEntity<Workloads>> getLegacyWorkloads(@RequestParam("stacks") String stacks) {
+    public Mono<ResponseEntity<Workloads>> getLegacyWorkloads(@RequestParam(value = "stacks", defaultValue = "", required = false) String stacks,
+    @RequestParam(value = "serviceOfferings", defaultValue = "", required = false) String serviceOfferings
```

Comment:
Start with `@RequestParam(value = "service-offerings"` because we want this to be consistent w/ README.md.